src/DragAndDropSupportWidget/widget/DroppableWidget.js

```diff
@@ -1,117 +1,67 @@
+], function (declare, _WidgetBase, lang, $) {
+    "use strict";
+
+    return declare("DragAndDropSupportWidget.widget.DroppableWidget", [ _WidgetBase ], {
+
+        // Modeler parameters
+        dropTargetClass: "",
+        dropTargetHoverClass: "",
+        dropTargetSelector: "",
+        acceptSelector: "",
+        dropEntityList: [],// [ draggedEntity, onDropEntity, draggedReference, dropTargetReference, onDropMicroflow ]
+
+        // Internal variables
         _contextObj: null,
         _dropData: null,
         _dropEntityItem: null,
 
-        // dojo.declare.constructor is called to construct the widget instance. Implement to initialize non-primitive properties.
-        constructor: function () {
-        },
-
-        // dijit._WidgetBase.postCreate is called after constructing the widget. Implement to do extra setup work.
         postCreate: function () {
-//            console.log(this.id + '.postCreate');
-            
-//            this.domNode.appendChild(dom.create('span', { 'class': 'DroppableWidget-message' }, this.messageString));
-
+            // console.log(this.id + '.postCreate');
             this._setupEvents();
         },
 
-        // mxui.widget._WidgetBase.update is called when context is changed or initialized. Implement to re-render and / or fetch data.
         update: function (obj, callback) {
-//            console.log(this.id + '.update');
-
+            // console.log(this.id + '.update');
             this._contextObj = obj;
-            this._resetSubscriptions();
-            this._updateRendering();
 
             callback();
         },
 
-        // mxui.widget._WidgetBase.enable is called when the widget should enable editing. Implement to enable editing if widget is input widget.
-        enable: function () {
-
-        },
-
-        // mxui.widget._WidgetBase.enable is called when the widget should disable editing. Implement to disable editing if widget is input widget.
-        disable: function () {
-
-        },
-
-        // mxui.widget._WidgetBase.resize is called when the page's layout is recalculated. Implement to do sizing calculations. Prefer using CSS instead.
-        resize: function (box) {
-
-        },
-
-        // mxui.widget._WidgetBase.uninitialize is called when the widget is destroyed. Implement to do special tear-down work.
-        uninitialize: function () {
-            // Clean up listeners, helper objects, etc. There is no need to remove listeners added with this.connect / this.subscribe / this.own.
-        },
-
         _setupEvents: function () {
-            var args = {},
-                node,
-                thisObj = this;
-            
-            if (this.acceptSelector) {
-                args.accept = this.acceptSelector;
+            var self = this;
+            // http://api.jqueryui.com/droppable/
+            var args = {
+                accept: this.acceptSelector || "*",
+                reedy: true,
```

Comment:
Are you sure this is correct? reedy i.s.o. greedy?

Reply:
Yes, see docs, http://api.jqueryui.com/droppable/#option-accept default `*`

Reply:
I suppose the *reedy* is just a typo then. Almost leaving for the airport. This will have to wait to mid August.